src/Datadog.Trace/ExtensionMethods/SpanExtensions.cs

```diff
@@ -67,6 +71,7 @@ public static void AddTagsFromDbCommand(this Span span, IDbCommand command)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Aren't those a bit big for inlining? 🤔

Reply:
Yeah, the attribute will not do anything here. Will remove it.

Reply:
done in 40744c8